tests/foreman/ui/test_activationkey.py

```diff
@@ -1290,11 +1288,45 @@ def test_delete_manifest(self):
 
         @Assert: Deleting a manifest removes it from the Activation key
 
-        @Status: Manual
-
-        @BZ: 1078676
-
         """
+        # Upload manifest
+        manifest_path = manifests.clone()
+        entities.Organization(
+            id=self.org_id,
+        ).upload_manifest(path=manifest_path)
+        # Create activation key
+        activation_key = entities.ActivationKey(
+            organization=self.org_id,
+        ).create()
+        # Associate a manifest to the activation key
+        for subscription in \
+                entities.Organization(id=self.org_id).subscriptions():
+            if subscription['product_name'] == DEFAULT_SUBSCRIPTION_NAME:
+                activation_key.add_subscriptions({
+                    'quantity': 1,
+                    'subscription_id': subscription['id'],
+                })
+                break
+        with Session(self.browser) as session:
+            session.nav.go_to_select_org(self.org_name)
+            # Verify subscription is assigned to activation key
+            self.navigator.go_to_activation_keys()
+            self.assertIsNotNone(
+                self.activationkey.search_key_subscriptions(
+                    activation_key.name, DEFAULT_SUBSCRIPTION_NAME)
+            )
+            # Delete the manifest
+            self.navigator.go_to_red_hat_subscriptions()
+            self.subscriptions.delete()
+            success = self.subscriptions.wait_until_element(
```

Comment:
Better to assertIsNotNone there, assertTrue have better meaning for boolean values.